client/src/components/FaucetForm.tsx

```diff
@@ -176,6 +242,16 @@ const FaucetForm = (props: any) => {
         setIsLoading(false);
     }
 
+    const getOptionByValue = (value: any) => {
+        let selectedOption: any = options[0];
+        options.forEach((option: any) => {
```

Comment:
Please add return values to all `forEach` statements. If there is no return value then use `void`.